icu4c/source/i18n/messageformat2_macros.h

```diff
@@ -46,6 +46,9 @@ using namespace pluralimpl;
 #define LOWERCASE_E ((UChar32)0x0065)
 #define UPPERCASE_E ((UChar32)0x0045)
 
+// The first three macro names are prefixed with DIGIT_
+// in order to avoid substituting the macros into the
+// definition of StandardPlural::Form (see standardplural.h)
```

Comment:
if these are needed they should be UMSG_ or similar prefix, or else constants in a namespace.

Reply:
Eliminated them in [1bdaa31](https://github.com/unicode-org/icu/pull/3063/commits/1bdaa3191bbc894d9ed3799c7f944c70216fb53f).